src/main/resources/static/common/js/common.js

```diff
@@ -718,8 +718,10 @@ function connectAndSubscribeMQ(role, $http, $rootScope, $filter, mqConsumeCallba
         method: 'GET',
         url: '/v1/MQConnectorUrl',
     }).then(function(result) {
-        var connectorUrl = result.data.data;
-        var ws = new WebSocket(connectorUrl, 'stomp');
+        let connectorUrl = result.data.data;
+        // When PDA is not deployed on localhost we need to use real IP.
+        connectorUrl = connectorUrl.replace("localhost", document.domain);
```

Comment:
`document.domain` is deprecated. [Document.domain - Web APIs | MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/domain)